libs/ledger/src/upow/synergetic_execution_manager.cpp

```diff
@@ -99,7 +100,7 @@ ExecStatus SynergeticExecutionManager::PrepareWorkQueue(Block const &current, Bl
     }
 
     // attempt to lookup the contract being referenced
-    auto it = work_map.find(node.contract_digest);
+    auto it = work_map.find(node.contract_digest.ToHex());
```

Comment:
I think that internal data structures should just store the binary version not the hex version

Reply:
Done.